daemon/libnetwork/internal/resolvconf/resolvconf.go

```diff
@@ -525,15 +525,11 @@ func removeInvalidNDots(options []string) []string {
 	return options[:n]
 }
 
-// errSystem implements [github.com/docker/docker/errdefs.ErrSystem].
-//
-// We don't use the errdefs helpers here, because the resolvconf package
-// is imported in BuildKit, and this is the only location that used the
-// errdefs package outside of the client.
-type errSystem struct{ error }
+// systemErr implements [github.com/docker/docker/errdefs.ErrSystem].
+type systemErr struct{ error }
```

Comment:
As CI is restarting; I pushed one more commit that made linting fail on BuildKit; only changing the name of the type, so should have no impact otherwise.